floors/validate.go

```diff
@@ -0,0 +1,58 @@
+package floors
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+func validateFloorRules(schema openrtb_ext.PriceFloorSchema, delimiter string, ruleValues map[string]float64) []error {
+	var errs []error
+	for key, val := range ruleValues {
+		parsedKey := strings.Split(key, delimiter)
+		if len(parsedKey) != len(schema.Fields) {
+			// Number of fields in rule and number of schema fields are not matching
+			errs = append(errs, fmt.Errorf("Invalid Floor Rule = '%s' for Schema Fields = '%v'", key, schema.Fields))
+			delete(ruleValues, key)
+			continue
+		}
+		delete(ruleValues, key)
```

Comment:
Since delete will always happen, should it be moved up before the if statement?

Reply:
yes